_posts/2023-03-06-notes-from-community-call-2 march-2023.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Notes from the Standard for Public Code community call - 6 March 2023"
+date: 2023-03-06
+author: Jan Ainali
+type: blogpost
+excerpt: Reflecting on the What this does not do sections
+categories:
+  - Community call
+---
+
+# 2 March 2023 Standard for Public Code community call notes
+
+## Attendees
+
+* [Jan Ainali](https://publiccode.net/who-we-are/team/jan-ainali.html)
+* [Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html)
+* Martin Dias d’Ullois
+* Johan Groenen
+
+## News from the Foundation for Public Code
+
+* New release for the Governance game, [version 1.0.2](https://github.com/publiccodenet/governance-game/releases/tag/1.0.2)
+* We will have a session at MozFest: [In 2030, governments only use open source: a thought experiment](https://schedule.mozillafestival.org/session/BRR3GV-1)
+
+## Notes
+
+When we recently changed the sections "Why this is important" to instead become an introduction to each criterion, we also noted that many of the sections "What this does not do" did not add a lot of value to the reader, and thus we created the issue [Review "What does this not do" sections #813](https://github.com/publiccodenet/standard/issues/813).
```

Comment:
```suggestion
When we recently changed the "Why this is important" section to instead become an introduction to each criterion, we also noted that many of the "What this does not do"  sections did not add a lot of value to the reader, and so we created the issue [Review "What does this not do" sections #813](https://github.com/publiccodenet/standard/issues/813).
```

Reply:
Is there a space too many before the newly added "sections"?